string state
int64 action
float32 reward
string next_state
---
bool update_status
